server/handler/eval_options.go

```diff
@@ -45,6 +45,11 @@ type PullEvaluationOptions struct {
 	//
 	// TODO(bkeyes): remove in version 2.0
 	Deprecated_AppName string `yaml:"app_name"`
+
+	// As of 2023-07-01 the commit.pushedDate graphql field is removed from GitHub.
+	// !Warning!
+	// Setting this option effectively breaks all usage of the invalidate_on_push approval rule
+	DoNotLoadCommitPushedDate bool `yaml:"do_not_load_commit_pushed_date"`
```

Comment:
Hi there! Thank you so much for this quick fix, my team and I really appreciate it. Will this option be available to set via an env var, similar to `shared_repository` (meaning something like `POLICYBOT_OPTIONS_DO_NOT_LOAD_COMMIT_PUSHED_DATE`)? We use env vars for our server configuration in our k8s deployment yaml.

Reply:
@agirlnamedsophia as far as I know to support this that would require a change in this piece of code to support it:
https://github.com/palantir/policy-bot/blob/fa1bb320a0c34797cb2b497c622208bd823b8eb4/server/handler/eval_options.go#L70-L75